src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,98 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double DEFAULT_GROWTH_OF_ARRAY = 1.5;
+    private Object[] elements = new Object[DEFAULT_CAPACITY];
+    private int size;
+
     @Override
     public void add(T value) {
-
+        if (size == elements.length) {
+            increaseCapacity();
+        }
```

Comment:
move `if` inside of this method

Reply:
not fixed, move if statement into increaseCapacity method

Reply:
Okay, i see that you already moved it into metho, so you no longer needed this if here, just remove it (same in other places where you call increase method)